src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -10,14 +10,16 @@ import ConsentBanner from '../containers/ConsentBanner';
 import getStatsDestination from '../contexts/RequestContext/getStatsDestination';
 import getStatsPageIdentifier from '../contexts/RequestContext/getStatsPageIdentifier';
 import getOriginContext from '../contexts/RequestContext/getOriginContext';
+import services from '../lib/config/services';
 
 const PageWrapper = ({ bbcOrigin, children, id, service, isAmp }) => {
   const env = process.env.APP_ENV;
   const { isUK, origin } = getOriginContext(bbcOrigin);
+  const { fonts } = services[service];
 
   return (
     <Fragment>
-      <GlobalStyle />
+      <GlobalStyle fonts={fonts} />
```

Comment:
If you move `<GlobalStyle fonts={fonts} />` to inside the children of `ServiceContextProvider`, `GlobalStyle` can use context to get the fonts, which I think is nicer than accessing the config directly